calculator_submit.rb

```diff
@@ -0,0 +1,58 @@
+# this version is an experiment with while loops and the .include method
+puts "Hi! Let's do math!"
+
+puts "Please tell me which kind of math you would like to do"
+math_type = gets.chomp.downcase
+
+#this array includes all valid math types
+#the while loop loops through the array with the .include method with the
+#argument of math_type as the paramenter
+#as long as the array.include?(math_type) is false the invalid input message
+#will print to the screen
+valid_math_types = ['+', 'add', 'addition', '-', 'sub', 'subtraction', '*',
+  'mult', 'multiply', 'multiplication', '/', 'div', 'divide', 'division', '^',
+  'exponent', 'exponential', '%', 'modulo', 'mod']
+  while valid_math_types.include?(math_type) == false
```

Comment:
This is an awesome readable condition for this while loop, I like that I can tell just from the names of your variables what will cause this loop to break.  You could even consider using the `!` operator, which makes it seem more readable to me, but at this point it's just personal preference:

`while !valid_math_types.include?(math_type)`